### DISCUSSION TITLE
OpenCrx Swagger playground - How To's

### DISCUSSION THREAD

**Discussion by javac9** [opencrx/opencrx-documentation#21] [Q&A]
@wfro64 A couple of questions regarding making `POST` requests in Swagger:

1. When making a request to create a `User`, there is also the `Contact` part, with `@href` and `$` symbols. I suppose for `@href` we can use the full `xri` string of an existing contact? What does `$` represent and what goes there?
2. Is it possible to send a `JSON object` for the `contact` part, instead of `xri` of an existing contact?

**Comment by wfro64** [comment]
See https://github.com/opencrx/opencrx-documentation/blob/master/Sdk/Rest.md#debugging how to set up a Swagger / REST playground.
